lib/cylc/gui/updater_graph.py

```diff
@@ -339,6 +369,9 @@ def update_graph(self):
         needs_redraw = current_id != self.prev_graph_id
 
         if needs_redraw:
+            self.vis_png_map = {}
+            self.regenerate_vis_png_map(self.state_summary)
```

Comment:
Do we need to do all the pngs again? Could we do something with the existing `vis_png_map` and `show_vis_tags` information?